GettingStartedDeb.md

```diff
@@ -125,3 +125,9 @@ The NuGet.config file should look something like the following:
 </configuration>
 ```
 The important part of this is that you have a package source with aspnetvnext and nuget.org in it.
+
+Nuget restore also checks the hash on the package. So ensure that you have `libcrypto` available. To do this run: 

```

Comment:
`libssl-dev` contains development libraries, header files, and manpages for `libssl`
 and `libcrypto`. So yes, it will suffice.

I am unaware if there is a more "compact" package without to acquire `libcrypto` with the additional stuff.